templates/customize-class/sortable-adventures.html

```diff
@@ -0,0 +1,37 @@
+<div class="flex flex-row w-full" id="adventure-dragger">
```

Comment:
Can we come up with a naming convention for files that are intentionally partials, and designed to be retrieved via HTMX?

In the past I did `hx-<name>.html`, but we can also go for `partial-<name>.html` or something.

Reply:
Sure! I'm going with your convention since it's the one already in place!

Reply:
I strongly prefer partial since it is a lot more descriptive of what it does, but let's merge this now, and I will make an issue separately since this PR is already open so long.